src/WebJobs.Extensions.DurableTask/DurableTaskExtension.cs

```diff
@@ -178,9 +181,10 @@ void IExtensionConfigProvider.Initialize(ExtensionConfigContext context)
 
             // Register the trigger bindings
             JobHostConfiguration hostConfig = context.Config;
+            ILogger logger = context.Config.LoggerFactory.CreateLogger(LoggerCategoryName);

```

Comment:
nit: You can use `LogCategories.CreateTriggerCategory("DurableTask")`, which will generate `Host.Triggers.DurableTask`. I made that function so we could keep things consistent. You're already doing that, but thought I'd point it out.

Reply:
Cool - I like consistency. :) I'll switch to `LogCategories.CreateTriggerCategory("DurableTask")`